pkg/osquery/tables/dev_table_tooling/table.go

```diff
@@ -0,0 +1,74 @@
+package dev_table_tooling
+
+import (
+	"context"
+	"encoding/base64"
+	"strings"
+
+	"github.com/go-kit/kit/log"
+	"github.com/go-kit/kit/log/level"
+	"github.com/kolide/launcher/pkg/osquery/tables/tablehelpers"
+	"github.com/osquery/osquery-go"
+	"github.com/osquery/osquery-go/plugin/table"
+)
+
+// Encapsulates the binary path(s) of a command allowed to execute
+// along with a strict list of arguments.
+type AllowedCommand struct {
+	binPaths []string
+	args     []string
+}
+
+type Table struct {
+	client    *osquery.ExtensionManagerClient
+	logger    log.Logger
+	tableName string
+}
+
+func TablePlugin(client *osquery.ExtensionManagerClient, logger log.Logger) *table.Plugin {
```

Comment:
As an FYI we're pretty inconsistent about whether we pass `client` to tables that don't need it. You can keep it, `_` it and not store it in the struct, or not pass it at all. (Some tables do need it, but most do not)